Providers/nxOMSContainers/containers/conf/container.conf

```diff
@@ -0,0 +1,141 @@
+# Fluentd config file for OMS Docker components
+
+# Forward port 25225 for container logs
+<source>
+	type forward
+	port 25225
+	bind 127.0.0.1
+</source>
+
+# Filter container logs
+<filter docker.**>
+	type filter_docker_log
+	log_path "/var/opt/microsoft/omsagent/log/filter_docker_log.txt"
```

Comment:
This may fall under the case you commented about [here](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/309/files/398ff0d2dd6396459bc6eee79a76348c54fb0c60#diff-2df9abbf7f5448ae92ad322e5ef43090), but if it's possible to make this a workspace-specific path, please do so. Otherwise, make a comment nearby that this will have to change in the future.